pootle/apps/import_export/utils.py

```diff
@@ -12,10 +12,14 @@
 from django.utils.translation import ugettext as _
 
 from pootle_store.models import Store
+from pootle_statistics.models import SubmissionTypes
 
 
-def import_file(file):
+def import_file(file, user=None):
     f = getclass(file)(file.read())
+    if not hasattr(f, "parseheader"):
+        raise ValueError(_("Unsupported filetype '%s', only po files " % (file.name)
+                           + "are supported at this time\n"))
```

Comment:
Please do not do this. Instead use:

```python
        raise ValueError(_("Unsupported filetype '%s', only PO files are "
                           "supported at this time.\n" % file.name))
```

Also I don't think that `file.name` actually provides the filetype.